chat-plugins/poll.js

```diff
@@ -5,11 +5,22 @@
 
 'use strict';
 
+/** @typedef {{source: string, supportHTML: boolean}} QuestionData */
+/** @typedef {{name: string, votes: number}} Option */
+
 class Poll {
+	/**
+	 * @param {ChatRoom} room
+	 * @param {QuestionData} questionData
+	 * @param {string[]} options
+	 */
 	constructor(room, questionData, options) {
+		// @ts-ignore
 		if (room.pollNumber) {
+			// @ts-ignore
```

Comment:
Apparently multiple games have need for a unique game ID. I think we might want to track them globally, so a `Rooms.gameNumber` could be nice.

Reply:
The thing is polls can't use gameNumber, because you can have a poll and hangman game running at the same time.